app/scripts/modules/core/src/widgets/spinners/spinner.component.ts

```diff
@@ -0,0 +1,9 @@
+import { module } from 'angular';
+import { react2angular } from 'react2angular';
+
+import { Spinner } from './Spinner';
+
+export const SPINNER = 'spinnaker.core.spinner.component';
```

Comment:
Should probably name this `SPINNER_COMPONENT` instead of `SPINNER` - would be a little more clear what it is, since we already have a `Spinner` in the `Spinner.tsx` file.

Reply:
Sounds good. will update.

Reply:
fixed.